pandas/tests/reshape/concat/test_concat.py

```diff
@@ -653,3 +653,30 @@ def test_concat_posargs_deprecation():
         result = concat([df, df2], 0)
     expected = DataFrame([[1, 2, 3], [4, 5, 6]], index=["a", "b"])
     tm.assert_frame_equal(result, expected)
+
+
+def test_concat_retain_attrs():
+    ''' Retain the attrs during concat
+
+    Only retain the attrs when the attrs are the same across all dataframes.'''
```

Comment:
Sorry, I should have mentioned that the quote char must be double quote, not single quote.
By the way, there is a detailed docstring guide https://pandas.pydata.org/pandas-docs/dev/development/contributing_docstring.html

Besides, it may be a good idea to remove docstrings and just post issue number ``# GH 41828``.

Reply:
@ivanovmg Thanks for the review. I have replaced the doc string with github issue.